### PR TITLE
New export part2

### PR DESCRIPTION
**Pull request by vincentfretin**
frontend:
- [x] appeler la mutation updateShareCount sur une idée ou un post

export_phase sheet:
- [x] Nombre de messages supprimés
- [x] Nombre de Top post
- [x] Nombre de messages (non top post)
- [x] Nombre de share de la thématique (need backend and frontend, we currently don't save this)
- [x] Nombre de share de message (need backend and frontend, we currently don't save this)
- [ ] Sentiment (analyse Watson)

survey, multicolumns, thread, brightmirror sheets: 
- [x] extra columns for sentiment actor
- [x] Nombre de share (need backend and frontend, we currently don't save this)
- [x] Message url / Fiction url  (use frontend_urls api)
- [ ] Sentiment (analyse Watson)

multicolumns sheet:
- [x] show column title instead of message_classifier (we don't know what is column1, column2)

thread sheet:
- [x] Message indentation (example 1.1.2)
- [x] Nombre de réponses
- [x] Titre du top post
- [x] Top post
- [x] Nombre de mots top post

brightmirror sheet:
- [x] Nombre de message (débat)
- [x] Nombre d'attrapages

### CODE REVIEW

**Comment by vincentfretin** [comment]
Somebody know what we need to do for the "Sentiment (analyse Watson)" column? Is watson sentiment really implemented?

File: assembl/tests/views_tests/test_routes.py [hunk 1]
```diff
@@ -494,6 +494,23 @@ def test_url_to_post_v2_proposal(discussion, proposals_en_fr,
...
+    expected = 'jacklayton2/debate/{}/question/{}/1/#{}'.format(
+        phase.identifier,
+        proposal.get_ideas()[0].graphene_id(),
+        proposals_en_fr[0].graphene_id())
     actual = frontend_urls.get_post_url(proposals_en_fr[0])
     assert expected in actual
+
+
+# this test fails with InvalidRequestError: Instance '<EmailAccount at 0x7fb21ceb4990>' is not persisted
+def xtest_url_to_fiction(discussion, post_published_for_bright_mirror):
```
Comment (ayyazdaniaryan) [inline assembl/tests/views_tests/test_routes.py]: Okay, can look into this.

File: assembl/lib/frontend_urls.py [hunk 2]
```diff
@@ -272,20 +273,30 @@ def get_post_url(self, post):
...
+                except ValueError:
+                    questionIndex = 1
+                route = self.get_frontend_url('questionPost', **{
                     'phase': phase.identifier,
                     'phaseId': phase.graphene_id(),
-                    'themeId': thematic.graphene_id(),
-                    'element': ''
+                    'questionId': first_idea.graphene_id(),
+                    'questionIndex': questionIndex,
+                    'element': post.graphene_id()
```
[the remark this answers is not in the document]
Comment (vincentfretin) [reply assembl/lib/frontend_urls.py]: All characters in a base64 string are already good for an url, there is no need to apply urlencode.

File: assembl/lib/frontend_urls.py [hunk 3]
```diff
@@ -272,20 +273,30 @@ def get_post_url(self, post):
...
-            else:
+            if not ideas:
                 # orphan post, redirect to home
                 return self.get_discussion_url()
 
-            if first_idea is not None and first_idea.__class__.__name__ ==\
-                    'Question':
-                thematic = post.get_closest_thematic()
-                route = self.get_frontend_url('post', **{
+            first_idea = ideas[0]
```
[the remark this answers is not in the document]
Comment (vincentfretin) [reply assembl/lib/frontend_urls.py]: I'm not sure I understand what you imply here. I didn't change anything here related to retrieving the first idea linked to the post. For the case of a PropositionPost (survey), the first idea is the question.

[the remark this answers is not in the document]
Comment (ayyazdaniaryan) [reply assembl/lib/frontend_urls.py]: That's exactly what I'm saying, at the API layer, we've made sure that interaction with the app does not allow for `PropositionPosts` to have `IdeaContentLinks` to bright mirror ideas, etc. Certainly it's possible for developers to change/write new code that might break this, if they are not aware of the assumptions. But honestly, I was just thinking out loud. Pulling the first idea from a `PropositionPost` works.

**Pull request merged**